CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+- Update from 2018 Rust Edtion to 2021
```

Comment:
Fwiw this only has meaning to code compiled within the crate, not to external crates consuming this, so there's no ~real~ reason to mention it in the changelog?

Reply:
This crate is pretty small and the changelog usally contains smaller things like Deps. updates, Many crates putting Editions changes in their changelog